data/json/items/armor.json

```diff
@@ -1063,13 +1063,6 @@
     "flags": [ "BLIND" ]
   },
   {
-<<<<<<< HEAD
-    "copy-from": "blindfold",
-    "material": [ "plastic", "cotton" ]
-  },
-  {
-=======
->>>>>>> parent of 6cc31694b9... Added blindfold_tape
```

Comment:
"What was that?"
"->>>>>>> parent of 6cc31694b9... Added blindfold_tape"
Those are messages about the difference between the two files. No idea how you managed to include those.

Reply:
By trying to revet to a previous commit.

Reply:
There should be a command for that, you shouldn't do that by hand, unless a command added that, which is even more confusing to me.

Reply:
It was done by the command...